Gitlab Event Collector (#19202)
demisto/content: Packs/GitLab/Integrations/GitLabEventCollector/GitLabEventCollector_description.md

> Gitlab Events collector XSIAM

Reviewer: ```suggestion
Gitlab Events collector
```

-
 Collects the events log for events and audit events provided by Gitlab API

---

* **Server URL** - The API domain URL for Okta.

> * **API key** - The request API key.

Reviewer: add link to documentation on how to generate the API key

> * **Event types** - The event ttpes to get as a comma delimitered string.

Reviewer: ```suggestion
* **Event types** - A comma-separated list of event types.
```
